src/scss/themes.scss

```diff
@@ -39,12 +39,17 @@ html {
     .tester-key.chatter-detected {
       background-color: rgb(233, 107, 107);
     }
-    #favorite-keyboard {
+    #favorite-keyboard,
+    #favorite-colorway {
       color: grey;
       &.active {
         color: $qmk-green;
       }
     }
+    #favorite-colorway {
```

Comment:
favorite-keyboard has pointer behavior already?

Reply:
does this belong in themes or in styles.scss?

Reply:
Oh yeah completely mixed those sheets